CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.2.3
+
+### Bug fixes
+
+* Fixed memory leak in `Java_io_realm_internal_Table_nativeSetPrimaryKey` (#3993).
```

Comment:
Minor, could this be rephrased to be more meaningful for users since `Java_io_realm_internal_Table_nativeSetPrimaryKey` is just one internal (native) method?